patterns/base/ul/pattern.yml

```diff
@@ -0,0 +1,12 @@
+name: Unordered List
+description: Unordered list of items
+twig: ./ul.twig
+sass: ./ul.scss
+category: base
+data:
+  ul:
+    class: base--ul
+  li:
+    class: base--li
+    text: List item 1
+    class: base--li
```

Comment:
or i can just hardcode the li values in to the twig template and then this wont be an issue..